user_stats/docker/requirements.txt

```diff
@@ -7,6 +7,8 @@ Django==4.2.7
 django-cors-headers==4.3.1
 gunicorn==21.2.0
 idna==3.4
+isort==5.13.2
```

Comment:
Remove this

Reply:
a7dac3f38c23597fb8d26752899a30b61ebfc107